### ISSUE TITLE
Unable to play files from cloud storage in 7.37-rc-2

### ISSUE THREAD

**Issue by reginabally** [Automattic/pocket-casts-android#902]
### Description

Reported in 6217246-zd-woothemes.

A user reported that they can't play their custom files in the cloud storage in the app version 7.37-rc-2. They were able to play the file on the web player and also on 7.36.

The following errors can be spotted in the logs:

```
I 23/4 14:39:45 Added episode to downloads. 21fda37e-49ed-4676-b63b-bc5d88a3462c podcast: null from: podcast download all
I 23/4 14:39:46 Worker Downloading episode file_example_MP3_5MG.mp3 21fda37e-49ed-4676-b63b-bc5d88a3462c
E 23/4 14:39:46 Invalid response returned for episode download. 400  https://api.pocketcasts.com/files/url/21fda37e-49ed-4676-b63b-bc5d88a3462c?token=AccessToken(value=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJwYzp0b2tlblR5cGUiOiJBQ0NFU1MiLCJwYzpwcm92aWRlciI6IlBPQ0tFVF9DQVNUUyIsInBjOnV1aWQiOiJhZDJkYTU1Mi04MDVhLTQzNGEtYjE5Yy1kNWUwNTRkYmFlMTUiLCJzY29wZXMiOlsibW9iaWxlIl0sImp0aSI6IjQ4ZmFjNzBlLWQ0MWMtNDAyNC04NWFhLTZhNzJiNTJjNzE2ZSIsImlhdCI6MTY4MjIzMTczMCwiZXhwIjoxNjgyMjM1MzMwLCJpc3MiOiJodHRwczovL2FwaS5wb2NrZXRjYXN0cy5jb20iLCJzdWIiOiJhZDJkYTU1Mi04MDVhLTQzNGEtYjE5Yy1kNWUwNTRkYmFlMTUifQ.HYIXlsmiak9msQ7Hc3h8PruUXbRTCAEXQxC6YQF7x1U)
E 23/4 14:39:46 Download failed file_example_MP3_5MG.mp3 21fda37e-49ed-4676-b63b-bc5d88a3462c - This episode may have been moved or deleted. Contact the podcast author. 
java.lang.RuntimeException: au.com.shiftyjelly.pocketcasts.repositories.download.task.DownloadEpisodeTask$DownloadFailed: This episode may have been moved or deleted. Contact the podcast author. 
	at tn.j.e(ExceptionHelper.java:14)
	at in.g.a(BlockingMultiObserver.java:34)
	at zm.b.h(Completable.java:9)
	at au.com.shiftyjelly.pocketcasts.repositories.download.task.DownloadEpisodeTask.u(DownloadEpisodeTask.kt:352)
	at androidx.work.Worker$a.run(Worker.java:3)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: au.com.shiftyjelly.pocketcasts.repositories.download.task.DownloadEpisodeTask$DownloadFailed: This episode may have been moved or deleted. Contact the podcast author. 
	at au.com.shiftyjelly.pocketcasts.repositories.download.task.DownloadEpisodeTask.F(DownloadEpisodeTask.kt:978)
	at au.com.shiftyjelly.pocketcasts.repositories.download.task.DownloadEpisodeTask.E(DownloadEpisodeTask.kt:56)
	at au.com.shiftyjelly.pocketcasts.repositories.download.task.DownloadEpisodeTask.v(Unknown Source:0)
	at j9.b.a(Unknown Source:2)
	at nn.b0.subscribeActual(ObservableCreate.java:11)
	at zm.p.subscribe(Observable.java:14)
	at nn.n0.subscribeActual(ObservableDoOnEach.java:18)
	at zm.p.subscribe(Observable.java:14)
	at nn.m1.y(ObservableIgnoreElementsCompletable.java:8)
	at zm.b.a(Completable.java:15)
	at zm.b.h(Completable.java:6)
	... 5 more
```

### Step-by-step reproduction instructions

1. Upload an mp3 file from the web player.
2. Install the app version 7.37-rc-2 on an Android device
3. Log in to the app and head to Profile > Files
4. Select the mp3 file to download
5. Notice the file wasn't downloaded

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

App version : 7.37-rc-2 (9086)
Phone: samsung - a53x - SM-A536B - a53x
Kernel: 5.10.136-android12-9-25898272-abA536BXXS5CWB6 - SWDK6311
Android version: 13 SDK 33

Reproduced on:
App version : 7.37-rc-2 (9086)
Phone: Google - emu64x - sdk_gphone64_x86_64 - emu64x
Kernel: 5.15.41-android13-8-00055-g4f5025129fe8-ab8949913 - abfarm-release-2004-0186
Android version: 13 SDK 33

**Comment by mchowning** [comment]
Recent PR to address this: https://github.com/Automattic/pocket-casts-android/pull/900

**Comment by ashiagr** [comment]
Fix (https://github.com/Automattic/pocket-casts-android/pull/900) is included in `7.37-rc-3`. Credits to @ecronin 🙇
